osbenchmark/workload/loader.py

```diff
@@ -1278,7 +1278,11 @@ def read(self, workload_name, workload_spec_file, mapping_dir):
                         ve.instance, indent=4, sort_keys=True),
                         ve.absolute_path, ve.absolute_schema_path))
 
-        current_workload = self.read_workload(workload_name, workload_spec, mapping_dir)
+        try:
+            current_workload = self.read_workload(workload_name, workload_spec, mapping_dir)
+        except Exception as e:
```

Comment:
Nit: We can use `WorkloadConfigError` from `exceptions.py` in place of the generic `Exception` class to be more descriptive. 
```
# from exceptions.py
class WorkloadConfigError(BenchmarkError):
    """
    Thrown when something is wrong with the workload config e.g. user supplied a workload-param
    that can't be set
    """

```

Reply:
> Nit: We can use `WorkloadConfigError` from `exceptions.py` in place of the generic `Exception` class to be more descriptive.
> 
> ```
> # from exceptions.py
> class WorkloadConfigError(BenchmarkError):
>     """
>     Thrown when something is wrong with the workload config e.g. user supplied a workload-param
>     that can't be set
>     """
> ```

The reason `Exception` was used here is to catch all possible errors that might be raised by `read_workload`.  For, instance, it might raise a `WorkloadSyntaxError`.  However, it's correct that a wrapped `WorkloadConfigError` could be raised once the exception is caught.  Will consider doing that in a future revision.